examples/jsm/loaders/KTX2Loader.js

```diff
@@ -242,6 +242,20 @@ class KTX2Loader extends Loader {
 				pvrtcSupported: renderer.extensions.has( 'WEBGL_compressed_texture_pvrtc' )
 					|| renderer.extensions.has( 'WEBKIT_WEBGL_compressed_texture_pvrtc' )
 			};
+			
+			if ( /Linux.*Gecko/g.test( navigator.userAgent ) &&
```

Comment:
Unfortunately I think this regex will also match for Chrome+Android on many devices. Most browsers include "Gecko" in the user agent string, and Android devices sometimes include "Linux". The `navigator.userAgent` string has... baggage. 🙃 

It should work to test that [`navigator.platform`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform) includes "Linux" and that `navigator.userAgent` includes "Firefox", though. We already rely on `isFirefox = userAgent.indexOf( 'Firefox' ) > - 1` in GLTFLoader, for example. Not aware of other tests for Linux in the codebase at the moment.

It would also be a good to test that `typeof navigator !== 'undefined'`, in case the loader is being used in Node.js or another JavaScript runtime.

Reply:
... looks like this literally changed just last month to not be deprecated anymore? https://github.com/mdn/browser-compat-data/pull/27627